teamcity: failed test: bank/zerosum-splits

The following tests appear to have failed on master (roachtest): acceptance/bank/zerosum-splits

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+bank/zerosum-splits).

[#970216](https://teamcity.cockroachdb.com/viewLog.html?buildId=970216):


```
acceptance/bank/zerosum-splits
--- FAIL: roachtest/acceptance/bank/zerosum-splits (4.520s)
	test.go:584,bank.go:379,bank.go:525,acceptance.go:65: pq: splits would be immediately discarded by merge queue; disable the merge queue first by running 'SET CLUSTER SETTING kv.range_merge.queue_enabled = false'





```

Please assign, take a look and update the issue accordingly.

--- a/pkg/cmd/roachtest/bank.go
+++ b/pkg/cmd/roachtest/bank.go
@@ -309,13 +309,9 @@ func (s *bankState) startSplitMonkey(ctx context.Context, d time.Duration, c *cl
 				client.RLock()
 				zipF := accountDistribution(r)
 				key := zipF.Uint64()
-				const splitQuery = `ALTER TABLE bank.accounts SPLIT AT VALUES ($1)`
 				c.l.Printf("round %d: splitting key %v\n", curRound, key)
-				_, err := client.db.Exec(`SET experimental_force_split_at = true`)
-				if err != nil && !testutils.IsSQLRetryableError(err) {
-					s.errChan <- err
-				}
-				_, err = client.db.Exec(splitQuery, key)
+				_, err := client.db.Exec(fmt.Sprintf(
+					`SET experimental_force_split_at = true; ALTER TABLE bank.accounts SPLIT AT VALUES (%d)`, key))
 				if err != nil && !(testutils.IsSQLRetryableError(err) || isExpectedRelocateError(err)) {
 					s.errChan <- err
 				}

